sbc_compassion/i18n/fr_CH.po

```diff
@@ -159,21 +159,21 @@ msgstr "Correspondant"
 #: model:ir.actions.act_window,name:sbc_compassion.action_compassion_sponsorship_correspondence_form
 #: model:ir.ui.menu,name:sbc_compassion.menu_sponsorship_correspondence
 #: view:recurring.contract:sbc_compassion.view_contracts_correspondence_form
-#: view:sponsorship.correspondence:sbc_compassion.view_sponsorship_correspondence_form
-#: view:sponsorship.correspondence:sbc_compassion.view_sponsorship_correspondence_tree
-#: view:sponsorship.correspondence:sbc_compassion.view_sponsorship_correspondence_tree_details
+#: view:correspondence:sbc_compassion.view_sponsorship_correspondence_form
+#: view:correspondence:sbc_compassion.view_sponsorship_correspondence_tree
+#: view:correspondence:sbc_compassion.view_sponsorship_correspondence_tree_details
 msgid "Correspondence"
 msgstr "Correspondance"
 
 #. module: sbc_compassion
-#: view:sponsorship.crosscheck:sbc_compassion.view_sponsorship_correspondence_crosscheck
+#: view:correspondence.template.crosscheck:sbc_compassion.view_sponsorship_correspondence_crosscheck
```

Comment:
I think it's ok but, I expected to "correspondence.crosscheck" instead of "correspondence.**template**.crosscheck